index.js

```diff
@@ -19,3 +21,155 @@ links.forEach((n) => n.addEventListener('click', () => {
+    description2: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.',
+    live: 'https://mumbereh.github.io/canopy-website',
+    project: 'https://github.com/mumbereh/canopy-website',
+  },
+  {
+    id: '03',
+    src: 'images/1.png',
+    title: 'Facebook 360',
+    description1: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.',
+    description2: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.',
+    live: 'https://mumbereh.github.io/Swampflix',
+    project: 'https://github.com/mumbereh/Swampflix',
+  },
+  {
+    id: '04',
+    src: 'images/2.png',
+    title: 'Uber Navigation',
+    description1: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.',
+    description2: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.',
+    live: 'https://mumbereh.github.io/Lambada',
+    project: 'https://github.com/mumbereh/Lambada',
+  },
+];
+
+// Select the card button and the popup container
+const cardBtn = document.querySelector('.card-btn');
+const popupContainer = document.querySelector('.popup');
+
+// Create a function to show the popup
+function showPopup() {
+  // Add a class to the popup container to show it
+  popupContainer.classList.add('show');
+
+  // Remove any existing content from the popup
+  popupContainer.innerHTML = '';
+
+  // Loop through the popups array and create the content for each popup
+  popups.forEach((popup) => {
+    const popupContent = `
+      <div class="popup">
+        <h2 class="title">${popup.title}</h2>
+        <ul class="field">
+          <li class="cano">CANOPY</li>
+          <li class="tag">&#8226; Back End Dev </li>
+          <li class="tag">&#8226; 2015</li>
+        </ul>
+        <img src="${popup.src}" alt="">
+        <div class="pop-para">
+          <p>${popup.description1}</p><br>
+          <p>${popup.description2}</p>
+        </div>
+        <div class="pop-skills">
+       <ul class="trainning2">
+        <li class="other-skills">HTML</li>
+        <li class="other-skills">CSS</li>
+        <li class="other-skills">JavaScript</li>
+        <li id="desk2" class="ruby">Ruby </li>
+        <li id="desk2" class="other-skills" >BootStrap</li>
+    </ul>
+    <a href="https://mumbereh.github.io/First-portfolio">
+    <button type="submit">See source<img class="search" src="icon/popup.png"></button>
+  </a>
+    <a href="https://github.com/mumbereh/First-portfolio">
+    <button type="submit">See source<img class="search" src="icon/popup.png"></button>
```

Comment:
- [ ] In addition, the popup I saw while I clicked the `see project` button of the first project, does not have the design matching the one given in the design to follow. It is way bigger than the screen and someone needs to scroll in other see other details which are different from what we expect as we see in the design given. Please edit your design in order to match the design given 😉 

Your design currently 👇 
![image](https://user-images.githubusercontent.com/90222110/233807831-0eb307fa-a33e-4b20-a7a9-a5c5e2778ee2.png)